docs/ref/signals.txt

```diff
@@ -406,6 +406,23 @@ Arguments sent with this signal:
 ``using``
     The alias of database on which a command will operate.
 
+``plan``
+    .. versionadded:: 1.10
+
+    The migration plan that is going to be used for the migration run. While
+    the plan is not public API, this allows for the rare cases when it is
+    critical to know the plan. A plan is a list of two-tuples with the first
+    item being the instance of a migration class and the second item showing
+    if the migration was rolled back (``True``) or applied (``False``).
+
+``apps``
+    .. versionadded:: 1.10
+
+    An instance of ``django.apps.registry.Apps`` containing the state of apps
```

Comment:
Yes but `StateApps` is a subclass of `Apps`. Since both are undocumented I felt like we should only mentions it quacks like `django.apps.apps`.